### PR TITLE
WMS ID: 11315 and WMS ID: 11496 - Update Instructions

### PR DESCRIPTION
**Pull request by ibalasun**
# Add here your WMS ID

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x] All filenames are lower case (including folders, images, files, etc.)
- [x] Filenames are descriptive
- [x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [x] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [x] Image references in markdown contain an alternate text

### CODE REVIEW

**Comment by ramumg** [comment]
Hi @ibalasun, 

Please make a pull request per WMS. 

Thank you!

**Comment by ibalasun** [comment]
Hi @ramumg ,

Updated. 

Thank you!

**Comment by denisemyrick** [comment]
Hi @ibalasun, please update branch. It's out of date with base. Thanks!

File: accessgovernancee2e/create-campaign/images/Screenshot 2024-02-20 at 11.40.17 AM.png
Comment (denisemyrick) [inline accessgovernancee2e/create-campaign/images/Screenshot 2024-02-20 at 11.40.17 AM.png:1]: Under create-changes you have a Screenshot image. File names need to be lowercase and that one needs to be renamed something more descriptive.

File: accessgovernance-access-controls/aginstance-cleanup/aginstance-cleanup.md
Comment (denisemyrick) [inline accessgovernance-access-controls/aginstance-cleanup/aginstance-cleanup.md:1]: Prerequisites is a requirement for all labs.

File: accessgovernancee2e/aginstance-cleanup/aginstance-cleanup.md
Comment (denisemyrick) [inline accessgovernancee2e/aginstance-cleanup/aginstance-cleanup.md:1]: All labs must have prerequisites

File: accessgovernancee2e/create-campaign/create-campaign.md
Comment (denisemyrick) [inline accessgovernancee2e/create-campaign/create-campaign.md:1]: While reviewing I noticed you're a little inconsistent with bolding next

File: accessgovernancee2e/deploying-using-terraform/deploying-using-terraform.md
Comment (denisemyrick) [inline accessgovernancee2e/deploying-using-terraform/deploying-using-terraform.md:1]: Needs prerequisites

**Review by denisemyrick — CHANGES_REQUESTED** [review]

File: accessgovernancee2e/deploying-using-terraform/deploying-using-terraform.md
Comment (ibalasun) [reply accessgovernancee2e/deploying-using-terraform/deploying-using-terraform.md:1]: Hi @denisemyrick ,

Closing this pull request. Will create a new one with the updated changes. Thanks

**Pull request closed without merging**